File: 프로그래머스/unrated/181836. 그림 확대/그림 확대.js
function solution(picture, k) {
    const result = []
    const row = picture.map(el => el.split('').map(pix => pix.repeat(k)).join(''));
    row.forEach(el => {
        for(let i = 0; i < k; i++){
            result.push(el)
        }
    })
    return result
}

File: 프로그래머스/unrated/181836. 그림 확대/그림 확대.py
def solution(picture, k):   
    row = list(map(lambda el: ''.join(list(map(lambda pix: ''.join(pix * k), list(el)))), picture));
    
    result = [];
    for i in row:
        for j in range(k):
            result.append(i);
    return result

    
               